src/helpers/discord_utils.ts

```diff
@@ -0,0 +1,204 @@
+                name: message.author.username,
+                icon_url: message.author.avatarURL()
+            },
+            title: `**${title}**`,
+            description: description
+        }
+    });
+}
+
+const sendOptionsMessage = async (message: Discord.Message, guildPreference: GuildPreference, db: Pool, updatedOption: string) => {
+    let cutoffString = `${guildPreference.getBeginningCutoffYear()}`;
+    let genderString = `${guildPreference.getSQLGender()}`;
+    let limitString = `${guildPreference.getLimit()}`;
+    let volumeString = `${guildPreference.getVolume()}`;
+
+    cutoffString = updatedOption == GameOptions.CUTOFF ? bold(cutoffString) : codeLine(cutoffString);
+    genderString = updatedOption == GameOptions.GENDER ? bold(genderString) : codeLine(genderString);
+    limitString = updatedOption == GameOptions.LIMIT ? bold(limitString) : codeLine(limitString);
+    volumeString = updatedOption == GameOptions.VOLUME ? bold(volumeString) : codeLine(volumeString);
+
+    let totalSongs = await getSongCount(guildPreference, db);
+    await sendInfoMessage(message,
+        updatedOption == null ? "Options" : `${updatedOption} updated`,
+        `Now playing the ${limitString} out of the __${totalSongs}__ most popular songs  by ${genderString} artists starting from the year ${cutoffString} at ${volumeString}% volume.`,
+        updatedOption == null ? `Psst. Your bot prefix is \`${guildPreference.getBotPrefix()}\`.` : null,
+        updatedOption == null ? "assets/tsukasa.jpg" : null
+    );
+}
+const getDebugContext = (message: Discord.Message): string => {
+    return `gid: ${message.guild.id}, uid: ${message.author.id}`
+}
+
+const getCommandFiles = (): Promise<{ [commandName: string]: BaseCommand }> => {
+    return new Promise((resolve, reject) => {
+        let commandMap = {};
+        fs.readdir("./commands", async (err, files) => {
+            if (err) {
+                reject();
+                return logger.error(`Unable to read commands error = ${err}`);
+            }
+            for (const file of files) {
+                let command = await import(`../commands/${file}`);
+                let commandName = file.split(".")[0];
+                commandMap[commandName] = new command.default()
+            }
+            resolve(commandMap);
+        });
+    })
+}
+
+const bold = (text: string): string => {
+    return `**${text}**`;
+}
+
+const italicize = (text: string): string => {
+    return `*${text}*`;
+}
+
+const codeLine = (text: string): string => {
+    return `\`${text}\``
+}
+
+const touch = (filePath: string) => {
+    try {
+        let currentTime = new Date();
+        fs.utimesSync(filePath, currentTime, currentTime);
+    } catch (err) {
+        fs.closeSync(fs.openSync(filePath, "w"));
+    }
+}
+
+const arraysEqual = (arr1: Array<any>, arr2: Array<any>): boolean => {
+    if (arr1.length !== arr2.length) {
+        return false;
+    }
+
+    var arr1 = arr1.concat().sort();
```

Comment:
Any reason why these can't be `let`s? Sorry, I know this was from something I had approved earlier, sorry about that.

Reply:
copied this from stackoverflow, fixed